ddtrace/mocktracer/mocktracer.go

```diff
@@ -192,3 +192,20 @@ func (t *mocktracer) Inject(context ddtrace.SpanContext, carrier interface{}) er
 	})
 	return nil
 }
+
+func (t *mocktracer) injectTextMap(context ddtrace.SpanContext, writer tracer.TextMapWriter) error {
```

Comment:
I think this function isn't used anywhere, right? We can drop it. It's dead code, unless I'm missing something.

Reply:
I really thought I had reverted all my changes to the mocktracer but obviously did not. Should be good now.